README.md

```diff
@@ -1 +1,189 @@
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+4. Run
+   ```
+   npx hint .
+   ```
+5. Fix validation errors.
+
+### [Stylelint](https://stylelint.io/)
+
+A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
+
+1. Run
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+not sure how to use npm? Read this.
+
+2. Copy .stylelintrc.json to the root directory of your project.
+
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+
+If you think that change is necessary - open a Pull Request in this repository and let your code reviewer know about it. 4. Run npx stylelint "\*_/_.{css,scss}" on the root of your directory of your project.
+
+5. Fix linter errors.
+
+6. **IMPORTANT NOTE:** feel free to research auto-correct options for Stylelint if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: (https://github.com/brighton0102)
+- Twitter: (@brighton_ui)
+- LinkedIn: (in/brighton-mtandi-976615267)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+      This project will not have any additional features.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly leave a comment below and share it with other web developers and lets enjoy this journey together.
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="Microverse Inc."></a>
+
+I am thankful to be part of the less than one percent of the student's who get enrolled at microverse.
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]**
+  Do I install linters on each and every project at microverse?
+
+  - [Answer_1]
+    Yes you have to use linters for each and every project at microverse.
+
+- **[Question_2]**
+    Should i always create a feature branch when working on projects?
+
+  - [Answer_2]
+    Yes you have to create a feature branch for your projects
+  <!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
- [ ] Please try as much as possible to customize the license to have your credentials. This will provide more project ownership and permission to other developers to contribute to your project ☺️.